src/test/java/it/gov/pagopa/payhub/auth/utils/AESUtilsTest.java

```diff
@@ -0,0 +1,41 @@
+package it.gov.pagopa.payhub.auth.utils;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.MockedStatic;
+
+import java.util.Base64;
+
+import static org.mockito.Answers.CALLS_REAL_METHODS;
+import static org.mockito.Mockito.mockStatic;
+
+public class AESUtilsTest {
```

Comment:
Why have you changed this test?
It was ok, while now this test has become useless (you are not more testing the intended method...
You are calling the method and verifying that you have called it inside the test itself...